### PR TITLE
added rpccorsdomain, logfile disabled for 1.4.0-unstable

### PR DESCRIPTION
**Pull request by euri10**
sorry for the .gitignore in the same pull
https://github.com/pipermerriam/populus/issues/83

### CODE REVIEW

**Comment by pipermerriam** [comment]
Can you add a new `click` command to the `chain_run` command something like the following so that it's configurable from the command line.

```python
@click.option(
     '--rpccorsdomain', default=None,
     help="""
     Determines the value that will be passed in as the `--rpcorsdomain` to the `geth` instance.
     """)
def chain_run(...):
    ...
```

File: populus/cli/chain_cmd.py [hunk 1]
```diff
@@ -63,11 +63,12 @@ def chain_run(name, mine, verbosity, active):
     ensure_account_exists(data_dir)
 
     kwargs = {
-        "logfile": logfile_path,
-        "verbosity": "%d" % verbosity
+        # "logfile": logfile_path,
+        "verbosity": "%d" % verbosity,
+        "rpccorsdomain": 'http://localhost:5000'
```
Comment (pipermerriam) [inline populus/cli/chain_cmd.py]: Have you tested this?  I believe the `http://` should be left off but I can't recall if that is the case or not.

**Comment by pipermerriam** [comment]
Also there are a few linting errors. https://travis-ci.org/pipermerriam/populus/jobs/124547131

**Comment by pipermerriam** [comment]
Once tests are passing this looks great.

**Comment by euri10** [comment]
added the click, changed the tox, and yest the http:// works fine. 1st time I do this so please apologize if I'm messing things up, git is rather hard for newbs like me :)

**Comment by euri10** [comment]
I think this fails test because I commented # "logfile": logfile_path,

**Comment by pipermerriam** [comment]
I appreciate the work you put into this.  Unfortunately, I don't believe that this pull request is viable given the significant changes that were just merged into master. 

If you would like to rework this to include the `--rpccorsdomain` changes to the `populus chain run` command I'd be happy to get that merged in.

**Pull request closed without merging**